Sources/SwerverCore/Router/Router.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
```

Comment:
Do you need to import `Foundation` here?

Reply:
fixed